### ISSUE TITLE
How much do they need?

### ISSUE THREAD

**Issue by singpolyma** [jayfk/fundingoss.com#29]
I would be awesome if, in addition to listing what funding projects have, we could source from them info about what they need. Do they need money? What order of magnitude? Do they need contributions? What kind? (code, review, gardening, design, other?)

**Comment by singpolyma** [comment]
Update: I know some of the above is covered by the "needs" tags, but not really the "how much" -- at least a ballpark or a goal they're working towards

**Comment by nayafia** [comment]
yeah. this is visible in the JSON under the open-ended questions "needs" and "funding status notes" (think it's tough to standardize "how much", so open-ended lets them structure it as they wish) but not currently part of the view on the site.

**Comment by jayfk** [comment]
Would it make sense to break this down into choices that can be selected, e.g:
- [ ] Developer
- [ ] Community Support
- [ ] Funding
- [ ] Code Reviews
- [ ] Maintainer

and leave the open-ended questions for further explanation?

**Comment by nayafia** [comment]
I like that idea!

**Comment by techtonik** [comment]
I will be interested to see the results of this survey, but my bet would be that `I need money to keep this project alive` would be the most popular option. Also, the question `how much` depends on accounting - it is rather hard to calculate the sum, and even harder to say that you need at least $666 per month to go on and this is [an absolute minimum](http://www.numbeo.com/cost-of-living/city_estimator?city=Minsk&country=Belarus&displayCurrency=USD&members=1&restaurants_percentage=5.0&inexpensive_restaurants_percentage=90.0&drinking_coffee_outside=30.0&going_out_monthly=2&smoking_packs_per_day=0.0&alcoholic_drinks=0.0&type_of_food=1&driving_car=0.0&taxi_consumption=0.0&paying_for_public_transport=Monthly%2C+All+Members&sport_memberships=1.0&vacation=25.0&clothing_and_shoes=25.0&rent=27&displayCurrency=USD&members=1&restaurants_percentage=5.0&inexpensive_restaurants_percentage=90.0&drinking_coffee_outside=30.0&going_out_monthly=2&smoking_packs_per_day=0.0&alcoholic_drinks=0.0&type_of_food=1&driving_car=0.0&taxi_consumption=0.0&paying_for_public_transport=Monthly%2C+All+Members&sport_memberships=1.0&vacation=25.0&clothing_and_shoes=25.0&rent=27).

**Comment by jayfk** [comment]
@techtonik yes, absolutely. My idea was to make it easier to get a quick overview of projects that are in need. If you are interested in a particular project, you will read the further explanation field anyway.
